test/ctia/entity/feed_test.clj

```diff
@@ -0,0 +1,377 @@
+   :relationships
+   [{:schema_version "1.0.11",
+     :target_ref
+     "transient:esa-indicator-ec95b042572a11894fffe553555c44f5c88c9199aad23a925bb959daa501338e",
+     :type "relationship",
+     :external_ids
+     ["ctia-feed-indicator-test"],
+     :short_description
+     "Judgement is part of indicator - 'Custom Malicious IP Watchlist'",
+     :title "judgement/indicator relationship",
+     :external_references [],
+     :source_ref
+     "transient:esa-judgement-4340e8cc49ff428e21ad1467de4b40246eb0e3b8da96caa2f71f9fe54123d500",
+     :id
+     "transient:esa-relationship-3c056c6ef8ace5057980b57f3eb07b916c84d94f7d1a340f41aba7630c459a5e",
+     :tlp "amber",
+     :timestamp "2019-05-08T18:03:32.785Z",
+     :relationship_type "element-of"}
+    {:schema_version "1.0.11",
+     :target_ref
+     "transient:esa-indicator-ec95b042572a11894fffe553555c44f5c88c9199aad23a925bb959daa501338e",
+     :type "relationship",
+     :external_ids
+     ["ctia-feed-indicator-test"],
+     :short_description
+     "Judgement is part of indicator - 'Custom Malicious IP Watchlist'",
+     :title "judgement/indicator relationship",
+     :external_references [],
+     :source_ref
+     "transient:esa-judgement-4340e8cc49ff428e21ad1467de4b40246eb0e3b8da96caa2f71f9fe54123d400",
+     :id
+     "transient:esa-relationship-3c056c6ef8ace5057980b57f3eb07b916c84d94f7d1a340f41aba7630c459a5d",
+     :tlp "amber",
+     :timestamp "2019-05-08T18:03:32.785Z",
+     :relationship_type "element-of"}
+    {:schema_version "1.0.11",
+     :target_ref
+     "transient:esa-indicator-ec95b042572a11894fffe553555c44f5c88c9199aad23a925bb959daa501338e",
+     :type "relationship",
+     :external_ids
+     ["ctia-feed-indicator-test"],
+     :short_description
+     "Judgement is part of indicator - 'Custom Suspicious IP Watchlist'",
+     :title "judgement/indicator relationship",
+     :external_references [],
+     :source_ref
+     "transient:esa-judgement-4340e8cc49ff428e21ad1467de4b40246eb0e3b8da96caa2f71f9fe54123d499",
+     :id
+     "transient:esa-relationship-4c056c6ef8ace5057980b57f3eb07b916c84d94f7d1a340f41aba7630c459a6d",
+     :tlp "amber",
+     :timestamp "2019-05-08T18:03:32.785Z",
+     :relationship_type "element-of"}]})
+
+(use-fixtures :once
+  (join-fixtures [mth/fixture-schema-validation
+                  helpers/fixture-properties:clean
+                  whoami-helpers/fixture-server]))
+
+(use-fixtures :each
+  whoami-helpers/fixture-reset-state)
+
+(defn feed-view-tests [feed-id feed]
+  (testing "GET /ctia/feed/:id/view?s=:secret"
+    (let [feed-view-url (:feed_view_url feed)
+          feed-view-url-csv (:feed_view_url_csv feed)
+          feed-view-url-csv-wrong-secret (->> (drop-last feed-view-url-csv)
+                                              (string/join ""))
+          response (client/get feed-view-url
+                               {:as :json
+                                :headers {"Authorization" "45c1f5e3f05d0"}})
```

Comment:
You should not define an authorisation header, I understood that this route should only require the secret